core/plugins/org.polarsys.capella.core.sirius.analysis/src/org/polarsys/capella/core/sirius/analysis/TitleBlockServices.java

```diff
@@ -197,7 +215,11 @@ public void createDiagramTitleBlock(DDiagram diagram) {
   public void createElementTitleBlock(DDiagramElement elementView, DDiagram diagram) {
     DAnnotation titleBlock = TitleBlockHelper.addElementTitleBlock(diagram, elementView);
     DAnnotation line = TitleBlockHelper.addTitleBlockLine(diagram, titleBlock);
-    TitleBlockHelper.addTitleBlockCell(diagram, line, DEFAULT_CELL_NAME, DEFAULT_CELL_CONTENT);
+    DAnnotation cell = TitleBlockHelper.addTitleBlockCell(diagram, line, DEFAULT_CELL_NAME, DEFAULT_CELL_CONTENT);
+    Object object = getCellContent(diagram, cell, diagram, titleBlock);

```

Comment:
rename cellContent?

Reply:
Accepted.

Reply:
Done. I gave up this line, no more needed to getCellContent.